packages/nameguard-js/src/secureprimaryName.test.ts

```diff
@@ -1,12 +1,23 @@
-import { describe, it, expect } from "vitest";
+import { describe, it, expect, beforeAll } from "vitest";
 import { createPublicClient, http } from "viem";
 import { mainnet } from "viem/chains";
 import { SecurePrimaryNameResult } from "@namehash/nameguard";
 import { securePrimaryName } from "./securePrimaryName";
+import { initializeData } from "./data";
 
 const PROVIDER_URI_MAINNET = process.env.PROVIDER_URI_MAINNET;
 
+if (!PROVIDER_URI_MAINNET) {
+  console.warn("PROVIDER_URI_MAINNET is not defined. Defaulting to viem's default provider, which may have rate limiting and other performance limitations.");
+}
+
+const TEST_TIMEOUT = 30000;
```

Comment:
```suggestion
const TEST_TIMEOUT_MS = 30000;
```

Is it correct to assume these are measured in milliseconds? Suggesting this change to make the unit of measurement more explicit.